helpers/cluster.js

```diff
@@ -14,7 +16,11 @@ const { storePageInTaskCache } = require('./task-cache');
 
 const sites = require('../sites');
 
+const userAgent = new UserAgent();
+
 puppeteer.use(StealthPlugin());
+puppeteer.use(Ua());
+
```

Comment:
Actually @labboy0276 it looks like this extra line break is causing an error with the linter. Mind removing it? Then this LGTM!